gcp_variant_transforms/vcf_to_bq.py

```diff
@@ -438,42 +454,64 @@ def run(argv=None):
       known_args.infer_annotation_types,
       counter_factory)
 
+  schema = schema_converter.generate_schema_from_header_fields(
+      header_fields, processed_variant_factory, variant_merger)
+
+  sharding = variant_sharding.VariantSharding(known_args.sharding_config_path)
+  if sharding.should_keep_shard(sharding.get_residual_index()):
+    num_shards = sharding.get_num_shards()
+  else:
+    num_shards = sharding.get_num_shards() - 1
+
+  if known_args.update_schema_on_append:
+    for i in num_shards:
+      table_suffix = sharding.get_output_table_suffix(i)
+      table_name = sample_info_table_schema_generator.compose_table_name(
+          known_args.output_table, table_suffix)
+      bigquery_util.update_bigquery_schema_on_append(schema.fields, table_name)
```

Comment:
I thought we couldn't update schema for existing tables? Is this the way we circumvent it?